node/cli_tools.go

```diff
@@ -48,6 +48,7 @@ func FlagsToArgs(flags Flags, basedir string, sepBase bool) ([]string, Metadata)
 
 	args := []string{
 		"--assertions-enabled=" + strconv.FormatBool(flags.AssertionsEnabled),
+		"--version=" + strconv.FormatBool(flags.Version),
```

Comment:
I don't think we should add the version flag into avash because it's just a flag to print the version of the node and exit immediately.

Reply:
I'm fine with keeping this if you think it's useful, but not sure if it's really necessary